go.mod

```diff
@@ -26,6 +26,7 @@ require (
 	github.com/docker/docker v20.10.7+incompatible
 	github.com/docker/go-metrics v0.0.0-20181218153428-b84716841b82 // indirect
 	github.com/docker/libtrust v0.0.0-20160708172513-aabc10ec26b7 // indirect
+	github.com/dzlab/elastic-go v0.0.0-20160522173446-355f7c99ab99
```

Comment:
Is this package compatible with es v5/v6/v7? If so, could we just replace all dependencies `github.com/elastic/go-elasticsearch`?

Reply:
`github.com/elastic/go-elasticsearch` belongs to the official dependencies. So it's more comprehensive. I referenced `github.com/dzlab/elastic-go` because of the error structure of the elasticsearch.  So I don't think it can replace that official package.